element_calcium_imaging/imaging.py

```diff
@@ -165,13 +165,36 @@ def make(self, key):
                     'data_path': [tiff_files[0].parent.as_posix()],
                     'tiff_list': [f.as_posix() for f in tiff_files]
                 }
-                
+
                 suite2p.run_s2p(ops=suite2p_params, db=suite2p_paths)  # Run suite2p
 
                 _, imaging_dataset = get_loader_result(key, ProcessingTask)
                 suite2p_dataset = imaging_dataset
                 key = {**key, 'processing_time': suite2p_dataset.creation_time}
 
+            elif method == 'caiman':
+                from element_interface.run_caiman import run_caiman
+
+                tiff_files = (ProcessingTask * scan.Scan * scan.ScanInfo * scan.ScanInfo.ScanFile & key).fetch('file_path')
+                tiff_files = [find_full_path(get_imaging_root_data_dir(), tiff_file) for tiff_file in tiff_files]
+                tiff_files = [f.as_posix() for f in tiff_files]
+
+                params = (ProcessingTask * ProcessingParamSet & key).fetch1('params')
+                sampling_rate = (ProcessingTask * scan.Scan * scan.ScanInfo & key).fetch1('fps')
+                output_dir = (ProcessingTask & key).fetch1('processing_output_dir')
+                output_dir = find_full_path(get_imaging_root_data_dir(), output_dir).as_posix()
+
+                ndepths = (ProcessingTask * scan.Scan * scan.ScanInfo & key).fetch1('ndepths')
+
+                is3D = True if ndepths > 1 else False
```

Comment:
```suggestion
                is3D = ndepths > 1
```

Reply:
@dimitri-yatsenko  I tried `ndepths > 1`before,  it returned `np.bool_` type, and `caiman` wasn't happy with that.

Reply:
Then `bool(ndepths > 1)`